ui/src/app/edge/settings/app/install.component.ts

```diff
@@ -161,7 +161,8 @@ export class InstallAppComponent implements OnInit, OnDestroy {
         }
 
         this.form.markAsPristine();
-        this.router.navigate(['device/' + (this.edge.id) + '/settings/app/']);
+        const navigationExtras = { state: { installedAnApp: true } };
```

Comment:
maybe replace `installedAnApp` with something which represents an installation and a deinstallation of an app maybe something like `appInstanceChange`